qt/python/mantidqt/project/test/test_plotssaver.py

```diff
@@ -206,6 +207,17 @@ def test_get_dict_from_fig_properties(self):
 
         self.assertDictEqual(return_value, expected_value)
 
+    def test_get_dict_from_fig_with_LogNorm(self):
```

Comment:
Do we not need another test for not LogNorm() - from a quick look it's not obvious that it's covered by the others